serializations.md

```diff
@@ -480,3 +481,23 @@ The channel offchain transaction is not included directly in the transaction tre
 , <closes_at>       , int()
 ]
 ```
+
+#### Proof of inclusion on state trees (POI)
+```
+[ {<accounts>  , [{binary(), [{binary(), [binary()]}]}]}
+, {<calls>     , [{binary(), [{binary(), [binary()]}]}]}
+, {<channels>  , [{binary(), [{binary(), [binary()]}]}]}
+, {<contracts> , [{binary(), [{binary(), [binary()]}]}]}
+, {<ns>        , [{binary(), [{binary(), [binary()]}]}]}
+, {<oracles>   , [{binary(), [{binary(), [binary()]}]}]}
+]
+```
+
+Each field is the representation of a subtree to the top level tree. The serialization should be interpreted as
+```
+{<tree>, [<proof_of_inclusion>]}
+where
+<proof_of_inclusion> := [{<root_hash>, [{<mpt_hash>, <mpt_value>}]}]
+```
+where `[{<mpt_hash>, <mpt_value>}]` is the sorted list of Merkel Patricia Tree nodes in the proof. If the subtree is empty,
```

Comment:
The serialization of `<mpt_value>`, that I understand being `[binary()]`, may benefit to be further specified. Though I consider it optional for this PR.